actix/src/sec-7-stream.md

```diff
@@ -1 +1,305 @@
-[WIP] 
\ No newline at end of file
+# Streams
+
+Actix allows the handling of Tokio Streams to act as a source of events
+or messages to actors. This is commonly used with networking servers
+and clients, for communication over TCP, UDP and UnixSockets.
+
+At it's core, Actix when using streams provides a mechanism for taking
+whole message frames from a stream, and converting these to messages which
+are sent to your Actor's handlers.
+
+To show how these parts are created, we'll create a TCP echo server.
+
+## Wrapping a stream of events
+
+A stream provides a series of frames to Actix which reperesent some event
```

Comment:
>some event

Not, event, it is just arbitrary values.
Stream items has no meaning on its own

See definition of `Stream` in futures:

>A stream of values, not all of which may have been produced yet.

>Stream is a trait to represent any source of sequential events or items which acts like an iterator but long periods of time may pass between items